includes/class-wp-job-manager-shortcodes.php

```diff
@@ -69,7 +69,16 @@ public function __construct() {
 	public function shortcode_action_handler() {
 		global $post;
 
-		if ( is_page() && has_shortcode( $post->post_content, 'job_dashboard' ) ) {
+		/**
+		 * Determine if the shortcode action handler should run.
+		 *
+		 * @param bool $should_run_handler Should the handler run.
+		 *
+		 * @return bool
+		 */
+		$should_run_handler = apply_filters( 'job_manager_should_run_shortcode_action_handler', is_page() && has_shortcode( $post->post_content, 'job_dashboard' ) );
```

Comment:
I wonder if we should include more context here? I guess there isn't much we can supply that isn't already accessible from globals. Any thoughts?